ndmg/register.py

```diff
@@ -266,6 +267,17 @@ def gen_tissue(self):
         print("Extracting brain from raw T1w image...")
         reg_utils.t1w_skullstrip(self.t1w, self.t1w_brain, self.skull)
 
+        #  QA part of skull strip
+        last_folder = self.namer.dirs["qa"]["base"]
+        QA_skullstrip_path = last_folder + '/skull_strip'
+        folder = os.path.exists(QA_skullstrip_path)
```

Comment:
remove `folder` variable

Reply:
fixed it in latest version

Reply:
![image](https://user-images.githubusercontent.com/55203069/70834631-d99f9380-1dc8-11ea-8c1b-80554628bc1f.png)